src/Runtime/Runtime.php

```diff
@@ -230,6 +230,27 @@ public function getEnvOption($key, $default = null)
     }
 
     /**
+     * Shortcut to get the the configuration option for a specific environment and merge it with
+     * the global one (environment specific overrides the global one if present).
+     *
+     * @param      $key
+     * @param null $defaultEnv
+     * @param null $defaultConfig
+     *
+     * @return array
+     */
+    public function getMergedEnvAndConfigOption($key, $defaultEnv = null, $defaultConfig = null)
```

Comment:
@Toflar please, rename the method to `getMergedOption` so it's not too bulky, and only use one $default.
Thanks!